### PR TITLE
Stylistic changes to the appearance of the nLab

### PR DESCRIPTION
**Pull request by DavidMichaelRoberts**
We are not running on the main branch of Instiki. This new colour is to create a visual distinction, following some private discussions.

### CODE REVIEW

**Comment by DavidMichaelRoberts** [comment]
Apologies, was confused as to the granularity of pull requests!
Also changed running font to now give a different overall feel to the pages.

**Comment by DavidMichaelRoberts** [comment]
Hi @williamsonrichard 

this is not a request, unfortunately, nor a mere tweak for the sake of it. These changes need to be made. You will find out soon, I think, why. Thank you for understanding

File: README [hunk 1]
```diff
@@ -1,4 +1,11 @@
```
[the remark this answers is not in the document]
Comment (DavidMichaelRoberts) [reply README]: OK, you can drop this one. Can we add a formal LICENSE file, though? We should have the text locally.

File: app/views/layouts/default.rhtml [hunk 2]
```diff
@@ -21,10 +21,10 @@
   <%= stylesheet_link_tag 'syntax', :media => 'all'  unless @inline_style %>
   <style type="text/css">
     h1#pageName, div.info, .newWikiWord a, a.existingWikiWord, .newWikiWord a:hover, [actiontype="toggle"]:hover, #TextileHelp h3 {
-      color: #<%= @web ? @web.color : "393" %>;
+      color: #<%= @web ? @web.color : "262" %>;
```
[the remark this answers is not in the document]
Comment (DavidMichaelRoberts) [reply app/views/layouts/default.rhtml]: OK, but this still should be changed, and the main nLab one should *also* be changed to match this colour.

File: public/stylesheets/instiki.css [hunk 3]
```diff
@@ -1,7 +1,7 @@
 body {
 background-color:#FFF;
 color:#333;
-font-family:Verdana, Arial, Helvetica, sans-serif;
+font-family:"Noto Sans", Roboto, sans-serif;
```
[the remark this answers is not in the document]
Comment (DavidMichaelRoberts) [reply public/stylesheets/instiki.css]: That's fine, I can't force people not to have generic sans-serif :-) Noto is an official Google font. I'm going to track down an iOS one too, and look at Windows/OSX fonts that can be added here. I will make a new pull request when I have tracked them down.

**Comment by DavidMichaelRoberts** [comment]
I do not know off-hand how to remove README from the pull request. Do I revert the file in my own fork?

**Comment by DavidMichaelRoberts** [comment]
OK, done it. The changes can now be merged and they should be pushed to the version that is actually running the nLab. The only question is where the main nLab web colour styling is stored. I guess this is changed through the Instiki interface on the actual website.

**Comment by DavidMichaelRoberts** [comment]
Oh, and Urs is now approving these changes as well.

File: README (hunk 1 above)
[the remark this answers is not in the document]
Comment (williamsonrichard) [reply README]: Will that work given that there are (at least) two different licenses involved? What is in the 'License' section currently is intended to be formal when it comes to my own code.

[the remark this answers is not in the document]
Comment (DavidMichaelRoberts) [reply README]: I tend to err on the side of a recognised license, though I know some people are so into freedom even this is too restrictive.

**Pull request merged**